app/models/service_credential_type.rb

```diff
@@ -0,0 +1,7 @@
+require "archived_concern"
+
+class ServiceCredentialType < ApplicationRecord
```

Comment:
There should be also `belongs_to :tenant and :source` and `acts_as_tenant(:tenant)`